composer/core/state.py

```diff
@@ -50,52 +50,43 @@ def null(precision):
 class State(Serializable):
     """The state of the trainer.
 
-    Contains variables that the trainer tracks throughout the training loop.
-    Note that the entire state is serialized when the trainer is checkpointed
-    so that it can be used restore the trainer and continue training from a
-    checkpoint. Algorithms are able to modify this object in-place.
+    Contains variables that the trainer tracks throughout the training loop.  Note that the entire state is serialized
+    when the trainer is checkpointed so that it can be used restore the trainer and continue training from a checkpoint.
+    :mod:`~composer.algorithms` are able to modify an instance of this class in-place.
 
 
     .. note::
 
-        To support multi-GPU training, :attr:`State.model` may be wrapped in :class:`DistributedDataParallel`,
-        and the dataloaders may be wrapped in a device-specific dataloader that handles moving tensors to device.
-
-    .. note::
-
-        ``Schedulers`` are wrapped in ``ComposableScheduler``, which handles stepping either stepwise or epochwise,
-        and also properly sets up learning rate warmups.
-
+        * To support multi-GPU training, ``State.model`` may be wrapped in :class:`torch.nn.parallel.DistributedDataParallel`,
+          and the dataloaders may be wrapped in a device-specific dataloader that handles moving tensors to device.
 
+        * :attr:`~.types.Schedulers` are wrapped in :attr:`~.scheduler.ComposerScheduler`, which handles stepping either
+          stepwise or epochwise, and also properly sets up learning rate warmups.
 
     Args:
-        model (types.Model, often ComposerModel): The model, typically as a subclass of :class:`ComposerModel`.
+        model (types.Model): The model, typically as a subclass of :class:`~.base.ComposerModel`.
```

Comment:
is it less verbose here to do :class:`.ComposerModel` ?

Reply:
Some of these I kept because they weren't linking correctly otherwise.